package.json

```diff
@@ -137,5 +133,8 @@
     "tsv": "^0.2.0",
     "typescript": "^4.8.4"
   },
-  "license": "Apache-2.0 OR MIT"
+  "license": "Apache-2.0 OR MIT",
+  "engines": {
+    "node": ">=20"
```

Comment:
Is this right? You haven't actually changed any runtime code.

Reply:
I thought of adding that since web streams, as globals, got added in Node.js 20

Reply:
Yes but you don't _have_ to use the globals - you can use the polyfills as before. None of the code (outside the tests) depends on those globals.